src/Processors/Learnosity/In/ValidationBuilder/ValidationBuilder.php

```diff
@@ -50,7 +52,12 @@ public static function build($questionType, $scoringType, array $responses)
                 $validResponseRef = new \ReflectionClass(self::BASE_NS . $questionType . '_validation_valid_response');
                 $validResponse = $validResponseRef->newInstance();
                 $validResponse->set_score($response->getScore());
-                $validResponse->set_value($response->getValue());
+                if($questionType == 'imageclozeassociationV2') {
+                   $convertArray = array_map(array(__CLASS__, 'convertArrayIntoArrayOfArray'), $response->getValue());
```

Comment:
This won't work like this - you'll get a `non-static method called statically` error. To do it this way you'll need to make `convertArrayIntoArrayOfArray` . Frankly though, this seems like an overly convoluted and indirect approach. Why not just do a for loop over the array and build a new array of arrays inline? If you'd rather abstract it then just pull the whole loop out into its own method. I don't see much reuse value in the `convertArrayIntoArrayOfArray` method as it is.

Reply:
Done